### PR TITLE
Clarify timer precision in docs

### PR DESCRIPTION
**Pull request by sapphonie**
Common footgun, should be clarified

### CODE REVIEW

**Review by peace-maker — CHANGES_REQUESTED** [review]
Even though this is an [implementation detail](https://github.com/alliedmodders/sourcemod/blob/6ebf04b75d82322f1e7729b3fbbdd645f77f9ec1/core/TimerSys.cpp#L39), it's a common source of confusion, so worth adding a note. Thank you!

File: plugins/include/timers.inc [hunk 1]
```diff
@@ -87,9 +87,10 @@ typeset Timer
 };
 
 /**
- * Creates a basic timer.  Calling CloseHandle() on a timer will end the timer.
+ * Creates a basic timer with 0.1 second accuracy.  Calling CloseHandle() on a timer will end the timer.
  *
  * @param interval      Interval from the current game time to execute the given function.
+ *                      Timer intervals are rounded to the nearest tenth of a second.
```
[the remark this answers is not in the document]
Comment (sapphonie) [reply plugins/include/timers.inc:93]: Honestly, after talking about it in the SM discord, I *still* don't quite understand what is going on with timer precision - as far as I know, it just executes on a think every 100ms, but a global think and not a per timer think?

**Comment by dvander** [comment]
It's driven by the framerate, so it's not super accurate or precise.On Jun 30, 2023 11:20 PM, sappho ***@***.***> wrote:
@sapphonie commented on this pull request.



In plugins/include/timers.inc:
>   *
  * @param interval      Interval from the current game time to execute the given function.
+ *                      Timer intervals are rounded to the nearest tenth of a second.

Honestly, after talking about it in the SM discord, I still don't quite understand what is going on with timer precision - as far as I know, it just executes on a think every 100ms, but a global think and not a per timer think?

**Review by Frenzoid — APPROVED** [review]

**Comment by sapphonie** [comment]
Closing in favor of #2014 , will reopen if that PR fails

**Pull request closed without merging**